streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/TopologyTest.scala

```diff
@@ -54,6 +42,7 @@ import scala.jdk.CollectionConverters._
  */
 //noinspection ScalaDeprecation
 @Timeout(600)
+@MockitoSettings(strictness = Strictness.STRICT_STUBS)
```

Comment:
There are two `TopologyTest` in code base, and the one which needs strict stub is `org.apache.kafka.streams.TopologyTest`

https://github.com/apache/kafka/blob/trunk/streams/src/test/java/org/apache/kafka/streams/TopologyTest.java#L83

Reply:
Thank you
I I have already added

Reply:
please revert those changes from this `TopologyTest` :)

Reply:
I'm sorry, I misunderstood before.